openreview/conference/templates/recruitReviewersProcess.js

```diff
@@ -1,26 +1,42 @@
 function() {
   var or3client = lib.or3client;
 
+  var SHORT_PHRASE = '';
+  var REVIEWER_NAME = '';
   var REVIEWERS_ACCEPTED_ID = '';
   var REVIEWERS_DECLINED_ID = '';
   var HASH_SEED = '';
 
   var hashKey = or3client.createHash(note.content.user, HASH_SEED);
 
   if (hashKey === note.content.key) {
+    var commonMessage = 'If you want to change your choice please click on the accept or decline links in the invitation email.';
     if (note.content.response === 'Yes') {
       console.log('Invitation replied Yes');
       //if a user is in the declined group, remove them from that group and add them to the reviewers group
+      var email = {
+        groups: [note.content.user],
+        subject: '[' + SHORT_PHRASE + '] You have accepted the ' + REVIEWER_NAME + ' invitation',
+        message: 'This email is to confirm that you have accepted the invitation to be a ' + REVIEWER_NAME + '.\n\n' + commonMessage
+      };
+
       or3client.removeGroupMember(REVIEWERS_DECLINED_ID, note.content.user, token)
       .then(result => or3client.addGroupMember(REVIEWERS_ACCEPTED_ID, note.content.user, token))
+      .then(result => or3client.or3request(or3client.mailUrl, email, 'POST', token))
       .then(result => done())
       .catch(error => done(error));
 
     } else if (note.content.response === 'No') {
       console.log('Invitation replied No');
+      var email = {
+        groups: [note.content.user],
+        subject: '[' + SHORT_PHRASE + '] You have declined the ' + REVIEWER_NAME + ' invitation',
+        message: 'This email is to confirm that you have declined the invitation to be a ' + REVIEWER_NAME + '.\n\n' + commonMessage
```

Comment:
Text suggestion -> 'You have declined the invitation to become a REVIEWER_NAME for SHORT_PHRASE. If you would like to change your decision, please click the Accept link in the previous invitation email.'

Maybe also say when the dealine for changing their decision is?